webapp/channels/src/components/channel_header_mobile/unmute_channel_button/unmute_channel_button.tsx

```diff
@@ -9,36 +9,28 @@ import {NotificationLevels} from 'utils/constants';
 
 type Actions = {
     updateChannelNotifyProps: (userId: string, channelId: string, props: ChannelNotifyProps) => void;
-}
+};
 
 type Props = {
     user: { id: string };
     channel: { id: string };
     actions: Actions;
-}
-
-export default class UnmuteChannelButton extends React.PureComponent<Props> {
-    handleClick = (): void => {
-        const {
-            user,
-            channel,
-            actions: {
-                updateChannelNotifyProps,
-            },
-        } = this.props;
-
-        updateChannelNotifyProps(user.id, channel.id, {mark_unread: NotificationLevels.ALL} as ChannelNotifyProps);
+};
+
+const UnmuteChannelButton = ({user, channel, actions}: Props) => {
+    const handleClick = () => {
+        actions.updateChannelNotifyProps(user.id, channel.id, {mark_unread: NotificationLevels.ALL} as ChannelNotifyProps);
```

Comment:
it is not really in the scope of this PR but can we get rid of typecasting `as ChannelNotifyProps`. 
You can change the `Actions` type from 
```ts
type Actions = {
     updateChannelNotifyProps: (userId: string, channelId: string, props: ChannelNotifyProps) => void;
 };
```
To:

```ts
type Actions = {
     updateChannelNotifyProps: (userId: string, channelId: string, props: Pick<ChannelNotifyProps, 'mark_unread'>) => void;
 };
```

Reply:
Done!